Formula/semgrep.rb

```diff
@@ -162,6 +162,7 @@ def install
       end
     end
 
+    ENV["SEMGREP_SKIP_BIN"] = "1"
```

Comment:
What is this line for?

Reply:
Outside homebrew setup.py assumes semgrep-core and spacegrep binaries are in specific locations and copy it over but since we build and bin install those two above we can skip that copying step.